<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminWindow</class>
 <widget class="QMainWindow" name="adminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管理员控制台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidgetUserList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户列表：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>历史记录：</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetHistory">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>481</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserIdHint">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelUserId">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>用户ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPointHint">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>积分：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPoint">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>积分</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>系统</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>管理员注销</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
